crates/astria-sequencer/src/transaction/signed.rs

```diff
@@ -32,7 +32,7 @@ use crate::{
 pub struct Signed {
     pub(crate) signature: Signature,
     pub(crate) public_key: VerificationKey,
-    pub(crate) transaction: Unsigned,
+    pub transaction: Unsigned,
```

Comment:
Please don't make this public and instead provide an accessor.

This change violates means that I can violate the invariant noted in the doc comment.